cocos/core/gfx/base/define.ts

```diff
@@ -1712,7 +1738,7 @@ export class QueryPoolInfo {
 
     constructor (
         public type: QueryType = QueryType.OCCLUSION,
-        public maxQueryObjects: number = 65536,
+        public maxQueryObjects: number = 32767,
```

Comment:
Half 16bit count should be 32768

Reply:
This is synced from native changes @stanleyljl